src/main/java/fi/metatavu/kuntaapi/server/integrations/ptv/translation/PtvTranslator.java

```diff
@@ -155,15 +156,48 @@ public ServiceLocationServiceChannel translateServiceLocationServiceChannel(
 
   public PrintableFormServiceChannel translatePrintableFormServiceChannel(
       PrintableFormServiceChannelId kuntaApiPrintableFormServiceChannelId, OrganizationId kuntaApiOrganizationId,
-      V7VmOpenApiPrintableFormChannel ptvPrintableFormServiceChannel) {
+      V8VmOpenApiPrintableFormChannel ptvPrintableFormServiceChannel) {
+    
+    if (ptvPrintableFormServiceChannel.getDeliveryAddresses() == null || ptvPrintableFormServiceChannel.getDeliveryAddresses().isEmpty()) {
+      return null;
+    }
+    
+    V8VmOpenApiAddressDelivery deliveryAddress = null;
+    List<V8VmOpenApiAddressDelivery> deliveryAddresses = ptvPrintableFormServiceChannel.getDeliveryAddresses().stream()
+      .filter(Objects::nonNull)
+      .filter((address) -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the parentheses around the "address" parameter [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1611)